cinema/urls.py

```diff
@@ -1,10 +1,30 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
+from cinema.views import (MovieViewSet, GenreList,
+                          GenreDetail, ActorList,
+                          ActorDetail, CinemaHallViewSet)
 
-from cinema.views import movie_list, movie_detail
+
+cinemahall_list = CinemaHallViewSet.as_view(
+    actions={"get": "list", "post": "create"})
+cinemahall_detail = CinemaHallViewSet.as_view(
+    actions={"get": "retrieve", "put": "update",
```

Comment:
it is better do this like that:
actions = {
    "get": "retrieve",
    "put": "update",
    "patch": "partial_update",
    "delete": "destroy"
}